scripts/orchestration/task_bootstrap.py

```
@@ -214,6 +228,140 @@ def _validated_judgment_activation(
+
+
+def _reconcile_requested_agent_dispositions(
+    *,
+    requested_agent_disposition: list[dict[str, str]],
+    primary_agent: str,
+    secondary_agents: list[str],
+    reviewer: str,
+) -> None:
+    """Align requested-agent disposition metadata with the final packet roles."""
+
+    advisory_statuses = {
```

Flake8 (7.3.0)

[error] local variable 'advisory_statuses' is assigned to but never used

(F841)